engine/access/ping/engine.go

```diff
@@ -107,18 +107,25 @@ func (e *Engine) startPing() {
 // pingNode pings the given peer and updates the metrics with the result and the additional node information
 func (e *Engine) pingNode(peer *flow.Identity) {
 	id := peer.ID()
+	var pingFailed bool
```

Comment:
```suggestion
	pingFailed := false
```
so readers don't need to know about the default bool value